site/content/en/docs/Guides/Client SDKs/csharp.md

```diff
@@ -85,11 +112,62 @@ This process is non-blocking internally.
 agonesSDK.WatchGameServer((gameServer) => { Console.WriteLine($"Server - Watch {gameServer}");});
 ```
 
+### Alpha: PlayerConnect

```

Comment:
Just realised one thing - let's make this match the REST API

Let's put a "Player Tracking" header of the top, and include the `{{< alpha title="Player Tracking" gate="PlayerTracking" >}}` standard shortcode, so we include the standard warning